### PR TITLE
fix app auto start issue

### PR DESCRIPTION
**Pull request by yhaspel**
App auto launch was not working properly. Fixed in this commit.

### CODE REVIEW

File: desktop/autoStartManager.js [hunk 1]
```diff
@@ -32,8 +32,7 @@ class WalletAutoStarter {
 
   isEnabled = async ({ event }) => {
     try {
-      const isEnabled = await this.walletAutoStarter.isEnabled();
-      store.set('isAutoStartEnabled', isEnabled);
+      const isEnabled = store.get('isAutoStartEnabled');
```
Comment (IlyaVi) [inline desktop/autoStartManager.js]: why did you change this?

Comment (yhaspel) [reply desktop/autoStartManager.js]: Value is already in store, and retrieving it here. Getting the value from walletAutoStarter seems to always return the same value, and does not work as expected. 
I've verified this with packaging the app, and restarting with the auto start feature on/off a few times.

Comment (IlyaVi) [reply desktop/autoStartManager.js]: Happy to see that you finally test the code, next step will be covering all possible conditions. For example - autostart is enabled. App is reinstalled. Store is empty. Settings screen show autostart is off - is this ok?

Comment (yhaspel) [reply desktop/autoStartManager.js]: Should it be on by default? If so, it's not ok and I'll change this.

Comment (IlyaVi) [reply desktop/autoStartManager.js]: As per electron app it doesn't come with any form of autostart, you have to check such things yourself before you make change to logic which you don't understand and submit it
